android/src/main/java/nl/fhict/happynews/android/activity/MainActivity.java

```diff
@@ -45,6 +50,13 @@ protected void onCreate(Bundle savedInstanceState) {
         loading = true;
```

Comment:
I think it would look nice if we can set the `SwipeRefreshLayout` to its `refreshing` state when we open the application.